features/polyExplorer/src/components/buttons/infoButton/infoButton.jsx

```diff
@@ -1,21 +1,20 @@
-import React from "react";
-import LinkButton from "../linkButton/linkButton.jsx";
+import React, { useContext } from "react";
+import { ExplorerContext } from "../../../context/explorer-context.jsx";
 import "./infoButton.css";
 
-const InfoButton = ({
-    infoScreenRoute,
-    saveActiveIndex,
-    stateChange = null,
-}) => (
-    <div onClick={saveActiveIndex}>
-        <LinkButton
-            className="info-button"
-            route={infoScreenRoute}
-            stateChange={stateChange}
-        >
+const InfoButton = ({ infoScreen, stateChange }) => {
```

Comment:
There's probably no chance at this point that you refactor this to use the `Burron` in poly-look, right? Maybe for the future?

Reply:
Big yes for the future. This is a now (1.5) solution. I think more work needs to be done regarding HTRT also build them from data, instead of having a component for each etc